src/Microsoft.Health.Fhir.Core/Features/Operations/IFhirOperationDataStore.cs

```diff
@@ -96,7 +96,7 @@ public interface IFhirOperationDataStore
         /// Queries the datastore for any reindex job documents with a status of running, queued or paused
         /// </summary>
         /// <param name="cancellationToken">The cancellation token</param>
-        /// <returns>True if any found</returns>
-        Task<bool> CheckActiveReindexJobsAsync(CancellationToken cancellationToken);
+        /// <returns>True if any found, along with id of the job</returns>
+        Task<(bool, string)> CheckActiveReindexJobsAsync(CancellationToken cancellationToken);
```

Comment:
>CheckActiveReindexJobsAsync [](start = 29, length = 27)

Discussion:
Is it worth to just return collection of currently running jobs instead of this limited tuple?
I personally find it's been a bit more cleaner in terms of how interface looks and how to work with it.

Reply:
For now just one job can be running, and I do like having the bool for a simple true/false comparison.

Reply:
nit: Can give these names: `(bool Found, string JobId)`. 

Though, having written that is feels more like the pattern:
```
Task<bool> TryGetReindexJobsAsync(out string jobId, CancellationToken cancellationToken);
```